src/docs/development/ios-project-migration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Xcode 11.4 Support
+description: How to migrate existing Flutter iOS projects to Xcode 11.4.
+---
+
+{{site.alert.note}}
+  You might be directed to this page if Flutter detects that your project
+  doesn't support Xcode 11.4.
```

Comment:
detects that you have Xcode 11.4 installed AND your project doesn't support Xcode 11.4?

Reply:
Yes, the tool gives the error message when the migration fails AND Xcode 11.4 or greater is installed.  We probably don't need to point that out, either they saw the error or they didn't.